app/workers/users/record_field_test_event_worker.rb

```diff
@@ -0,0 +1,31 @@
+module Users
+  class RecordFieldTestEventWorker
+    include Sidekiq::Worker
+    include FieldTest::Helpers
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(user_id, experiment, goal)
+      return if user_id.blank?
```

Comment:
we don't need this is f we use `after_create_commit`

Reply:
This was actually to guard against pageviews which have no user_id because they're used for use and anonymous pageview, but I just realized I also used it in the model, so can be removed here.